app/Enums/Gender.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace App\Enums;
+
+use BenSampo\Enum\Enum;
+
+/**
+ * @method static static OptionOne()
+ * @method static static OptionTwo()
+ * @method static static OptionThree()
+ */
+final class Gender extends Enum
+{
+    const male = "男";
```

Comment:
定数には数字や英単語の文字列を指定するのが一般的です。
プログラムは基本的に数字や英語で書くので、漢字やひらがななど日本語の文字列を書くと読みにくいのに加え、
日本語の文字列は文字コードの違いによるバグが起きやすいリスクもあります。

Reply:
```suggestion
    const Male = 'male';
    // または
    const MALE = 'male';
```
定数は大文字やアッパーキャメルケースで書いてください。

Reply:
定数名を大文字に、値は数値に修正致しました。
ご確認よろしくお願い致します。
app/Enums/Prefectures.php
app/Enums/Gender.php
app/Enums/UserStatus.php